app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/tasklist/multiple_indirect_disposals_task_list.scala.html

```diff
@@ -0,0 +1,182 @@
+        messages("task-list.indirect.enter-example-property-address.link")
+    else
+        messages("task-list.enter-example-property-address.link")
+}
+
+@exemptionsAndLossesState = @{
+    exampleCompanyDetailsState match {
+        case Complete =>
+            draftReturn.exemptionAndLossesAnswers.fold[TaskListStatus](ToDo)(_.fold(_ => InProgress, _ => Complete))
+        case _ =>
+            CannotStart
+    }
+}
+
+@ytdLiabilityState = @{
+    exemptionsAndLossesState match {
+        case Complete =>
+            draftReturn.yearToDateLiabilityAnswers.fold[TaskListStatus](ToDo) {
+                case c: CalculatedYTDAnswers => c.fold(_ => InProgress, _ => Complete)
+                case n: NonCalculatedYTDAnswers => n.fold(_ => InProgress, _ => Complete)
+            }
+        case _ => CannotStart
+    }
+}
+
+@uploadSupportingDocsState = @{
+    draftReturn.supportingEvidenceAnswers.fold[TaskListStatus](ToDo)(_.fold(_ => InProgress, _ => Complete))
+}
+
+@checkAndSendReturnState = @{
+    (exampleCompanyDetailsState, exemptionsAndLossesState, ytdLiabilityState, uploadSupportingDocsState) match {
+        case (Complete, Complete, Complete, Complete) => ToDo
+        case _                                        => CannotStart
+    }
+}
+
+
+@mainTemplate(title = title, userType = request.userType) {
+
+    @pageHeading(title)
+
+    <ol class="app-task-list">
+        <li>
+            <h2 class="app-task-list__section">
+                <span class="app-task-list__section-number">
+                    1.
+                </span>
+                @messages("triage.caption")
+            </h2>
+
+            <ul class="app-task-list__items">
+
+                @representeeState.map { state =>
+                    <li id="representee" class="app-task-list__item">
+                        @taskLink(messages("task-list.representee.link"), returns.representee.routes.RepresenteeController.checkYourAnswers().url, state)
+                    </li>
+                }
+
+                @if(!representeeState.forall(_.isComplete)) {
+                </ul>
+        @warning(messages("task-list.incompleteTriage"))
+        } else {
+        <li id="canTheyUseOurService" class="app-task-list__item">
+        @taskLink(messages("task-list.triage.link"), returns.triage.routes.MultipleDisposalsTriageController.checkYourAnswers().url, triageState)
+        </li>
+        </ul>
+
+
+    @if(!triageState.isComplete) {
+        @warning(messages("task-list.incompleteTriage"))
+    } else {
+        <li>
+            <h2 class="app-task-list__section">
+                <span class="app-task-list__section-number">
+                    2.
+                </span>
+                @messages("task-list.provide-details.heading")
+            </h2>
+            <ul class="app-task-list__items">
+                <li id="examplePropertyDetails" class="app-task-list__item">
+                @taskLink(
+                    examplePropertyDetailsTaskName,
```

Comment:
shouldn't this be something like `Enter company details` instead of something like `Enter example property details`?

Reply:
unless you were gonna do that in another PR...

Reply:
Yes it is displaying "Enter company details" only. But as its always for multiple indirect tasklist, no need to do this:
```@examplePropertyDetailsTaskName = @{
    if(draftReturn.isMultipleIndirectDisposal)
        messages("task-list.indirect.enter-example-property-address.link")
    else
        messages("task-list.enter-example-property-address.link")
}```

just use this 

``` 
<li>
            <h2 class="app-task-list__section">
                <span class="app-task-list__section-number">
                    2.
                </span>
                @messages("task-list.provide-details.heading")
            </h2>
            <ul class="app-task-list__items">
                <li id="examplePropertyDetails" class="app-task-list__item">
                @taskLink(
                    messages("task-list.indirect.enter-example-property-address.link"),
                    returns.address.routes.PropertyDetailsController.checkYourAnswers().url,
                    exampleCompanyDetailsState
                )
                </li>
            </ul>
        </li>
```